eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -172,15 +172,18 @@ jobs:
               TestPipeline: ${{ parameters.TestPipeline }}
 
         - pwsh: |
-            $setupFile="doxygen-1.8.18-setup.exe";
+            $setupFile='doxygen-1.9.7-setup.exe';
 
             Invoke-WebRequest `
               -MaximumRetryCount 10 `
-              -Uri "https://azuresdktooling.blob.core.windows.net/dependencies/$setupFile" `
+              -Uri "https://github.com/doxygen/doxygen/releases/download/Release_1_9_7/$setupFile" `
```

Comment:
Will this link be reliable (in general)?

Reply:
QQ: Do you trust the [doxygen distribution page](https://www.doxygen.nl/download.html)? It says that 1.9.7 is found at this location.

Reply:
Good point @ahsonkhan... I trust that the file will be delivered over HTTPS and so will match what's been uploaded to the release... but putting a copy of the file in our own storage will prevent those bits from changing out from under us. 

It's possible that the specific file in the release we're downloading could change without notification. All other engsys risks are assumed. 

Other options include: 

* Checking a hash of the file downloaded from GitHub 
* Installing by other means (like WinGet) 

It is nice that Doxygen has upgraded to serving files over HTTPS since we last set this up as we won't have to search for a trusted source from which to download new versions.